dbms/src/Server/Server.cpp

```diff
@@ -1325,6 +1326,24 @@ int Server::main(const std::vector<std::string> & /*args*/)
         settings.max_memory_usage_for_all_queries.getActualBytes(server_info.memory_info.capacity),
         settings.bytes_that_rss_larger_than_limit);
 
+    if (global_context->getSharedContextDisagg()->isDisaggregatedComputeMode())
+    {
+        // No need to have local index scheduler.
+    }
+    else if (global_context->getSharedContextDisagg()->isDisaggregatedStorageMode())
+    {
+        global_context->initializeGlobalLocalIndexerScheduler(
```

Comment:
Can you explain this math?

Reply:
initialize the LocalIndexerScheduler introduce by https://github.com/pingcap/tiflash/pull/9383, all vector index build task will be schedule by this scheduler.

Reply:
I checked that PR, and I can not figure out why it is `* 8 / 10`

Reply:
```
// There is no compute task in write node.
// Set the pool size to 80% of logical cores and 60% of memory
// to take full advantage of the resources and avoid blocking other tasks like writes and compactions.
```

Add some comments.